test/recipes/fuzz.pl

```diff
@@ -9,23 +9,23 @@
 use warnings;
 
 use OpenSSL::Glob;
-use OpenSSL::Test qw/:DEFAULT srctop_file/;
+use OpenSSL::Test qw/:DEFAULT srctop_dir/;
 
-sub fuzz_tests {
-    my @fuzzers = @_;
+sub fuzz_test {
```

Comment:
Actually, now that it takes just one argument and we know it only takes one argument and only does one thing, this function can be greatly reduced:

``` perl
sub fuzz_test {
    die "Only one argumented accepted" if scalar @_ != 1;

    my $f = $_[0];
    my $d = srctop_dir('fuzz', 'corpora', $f);
    subtest "Fuzzing $f" => sub {
        plan skip_all => "No directory fuzz/corpora/$f" unless -d $d;
        plan tests => 1;
        ok(run(fuzz(["$f-test", $_])));
    };
}
```

But even that seems a bit overly involved, it could be further reduced to this:

```
sub fuzz_test {
    die "Only one argumented accepted" if scalar @_ != 1;

    my $f = $_[0];
    my $d = srctop_dir('fuzz', 'corpora', $f);

    return run(fuzz(["$f-test", $d])) if -d $d;

    # Directory $d doesn't exist, so write a diagnostic and return false
    diag("No directory fuzz/corpora/$f");
    return 0;
}
```

... and with that, you can change all the `fuzz_test($fuzzer)` lines with this:

``` perl
ok(fuzz_test($fuzzer), "Fuzzing $fuzzer");
```

Reply:
See 55a3d207bb